openapi/components/schemas/Applications/ApplicationInstance.yaml

```
@@ -12,16 +12,21 @@ properties:
       - enabled
       - disabling
       - disabled
-  settings:
+  isConfigured:
     description: |-
-      Settings that are available for the user to complete when they install the application.
-      Use this object to describe fields that are rendered during installation, in the settings form.
+      Determines if the application instance configuration complete.
+      Configuration is done by [PutApplicationInstanceConfiguration](https://all-rebilly.redoc.ly/tag/Application-users/operation/PutApplicationInstanceConfiguration).
+    type: boolean
+    readOnly: true
+  settings:
+    description: Settings that are available for the user to complete when they install the application.
     type: object
     additionalProperties:
       type: string
```

suggestion: probably good to describe this field also (I'm not sure if my suggestion is correct)
```suggestion
      type: string
      description: Additional properties that the user may want to add when installing an application.
```